plugins/lookup/aws_secret.py

```diff
@@ -49,6 +49,15 @@
         - No effect when used with I(bypath).
     type: boolean
     default: false
+  on_deleted:
+    description:
+        - Action to take if the secret has been marked for deletion.
+        - C(error) will raise a fatal error when the secret has been marked for deletion.
+        - C(skip) will silently ignore the deleted secret.
+        - C(warn) will skip over the deleted secret but issue a warning.
+    default: error
+    type: string
+    choices: ['error', 'skip', 'warn']
```

Comment:
@tremble Thank you. `version_added: 2.0`?

Reply:
version_added, added